app/uk/gov/hmrc/helptosavefrontend/connectors/SessionCacheConnector.scala

```diff
@@ -21,6 +21,7 @@ import com.google.inject.{ImplementedBy, Inject, Singleton}
 import play.api.libs.json.{Reads, Writes}
```

Comment:
are we not doing metrics for other connectors  ?

Reply:
there's no reason not to - I'll stick them in there as well then